.github/workflows/main.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         with:
           version: "10.0"
           directory: ${{ runner.temp }}/llvm
-      
+
```

Comment:
Please avoid format change to avoid unnecessary diff when submitting PR to upstream